game/src/sandbox/misc_tools.rs

```diff
@@ -12,7 +12,7 @@ use crate::game::{DrawBaselayer, State, Transition};
 use crate::render::{DrawOptions, BIG_ARROW_THICKNESS};
 
 pub struct RoutePreview {
-    preview: Option<(AgentID, Time, Drawable)>,
+    preview: Option<(AgentID, Time, bool, Drawable)>,
```

Comment:
clarity nit: I could guess what most of these other values are, but `bool` is pretty opaque - might be time for a struct with named params.

Reply:
Given this is a very tightled scope tuple, how do you feel about the comments added?

Reply:
yep, that help!